microservices/netinfra_utils/workflows/Get_all_devices_as_dynamic_fork_tasks_from_inventory.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Get_all_devices_as_dynamic_fork_tasks_from_inventory",
```

Comment:
better name would be Get_all_devices_from_inventory_as_dynamic_fork_tasks
the same naming change for others please